modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -102,6 +104,12 @@ namespace imgproc {
         }
     };
 
+    G_TYPED_KERNEL(GGoodFeatures, <GArrayPoint2f(GMat,int,double,double,Mat,int,bool,double)>, "org.opencv.imgproc.goodFeatures") {
+        static GArrayDesc outMeta(GMatDesc, int, double, double, Mat, int, bool, double) {
```

Comment:
Mat -> const cv::Mat&

Reply:
OK.
But want to draw attention that there are other kernels in the file (and I'm afraid not only in that one) where outmeta parameters are not const. Should I create a task on that issue?

Reply:
Yes please.